File: appdeck-ios/AppDeck/AppDeck-lib/Ad/SmartAdServer/SmartAdServer-5.0/Samples/ObjCSample/ObjCSample/MenuItem.m
//
//  MenuItem.m
//  ObjCSample
//
//  Created by Loïc GIRON DIT METAZ on 15/01/13.
//  Copyright (c) 2013 Smart AdServer. All rights reserved.
//

#import "MenuItem.h"


@implementation MenuItem

- (id)initWithTitle:(NSString *)title {
	self = [super init];
	
	if (self) {
		_title = title;
	}
	return self;
}

@end

File: appdeck-ios/AppDeckTestApp/AppDeck-lib/Ad/SmartAdServer/SmartAdServer-4.5/Samples/SwiftSample/SwiftSample/MenuItem.swift
//
//  MenuItem.swift
//  SwiftSample
//
//  Created by Loïc GIRON DIT METAZ on 22/07/2014.
//  Copyright (c) 2014 Smart AdServer. All rights reserved.
//


// Represents an entry in the Main menu.
class MenuItem: NSObject {
	
	let title: String
	let segueIdentifier: String
	
	init(title: String, segueIdentifier: String) {
		self.title = title
		self.segueIdentifier = segueIdentifier
		super.init()
	}
	
}